### PR TITLE
Added location to map, added 2023 badge

### PR DESCRIPTION
**Pull request by ClJarvis**
## Linked Issue
issue #995

## Description
Added self to map.
added 2023 badge
removed twitter link.

## Code of Conduct

> By submitting this pull request, you agree to follow our [Code of Conduct](https://virtualcoffee.io/code-of-conduct/)

### CODE REVIEW

File: members/members/ClJarvis.ts [hunk 1]
```diff
@@ -47,4 +47,10 @@ export const ClJarvis: MemberObject = {
 		},
 	],
 	badges: ['Hacktoberfest2022'],
+	['Hacktoberfest2023'],
```
Comment (danieltott) [inline members/members/ClJarvis.ts:50]: Let's make this one array with two strings 👍

```suggestion
	badges: ['Hacktoberfest2022', 'Hacktoberfest2023'],
```

**Review by danieltott — CHANGES_REQUESTED** [review]
@ClJarvis just a small tweak to get this going

**Review by juliaseid — CHANGES_REQUESTED** [review]
Quick fix needed! 

Lines 49 & 50 - `badges` should be an array of strings, `['Hacktoberfest2022', 'Hacktoberfest2023']`

Dan said I could change this myself but I can't figure out how!

**Review by juliaseid — APPROVED** [review]

**Review by juliaseid — APPROVED** [review]

**Review by juliaseid — APPROVED** [review]

**Review by danieltott — CHANGES_REQUESTED** [review]
Thanks Chris!

**Pull request merged**